examples/check-if-image-loaded/README.md

```diff
@@ -0,0 +1,20 @@
+# Check If an Image Has Loaded Without Errors
```

Comment:
`# Check If an Image Has Loaded`

Reply:
`Loaded Without Errors`. 

It's a tautology.

If image is loaded when there is no errors are occurfed during its loading.
Remove `without errors` below.

Reply:
edited

Reply:
edited